components/sup/Cargo.toml

```diff
@@ -28,6 +28,7 @@ habitat_core = { path = "../core" }
 habitat_depot_client = { path = "../builder-depot-client" }
 habitat-eventsrv-client = { path = "../eventsrv-client" }
 habitat-launcher-client = { path = "../launcher-client" }
+habitat-launcher-protocol = { path = "../launcher-protocol" }
```

Comment:
@mwrock I hear you. My goal here is to appropriate a pattern through our crates where the consumer of a client doesn't need to understand the protocol at all

It's worth noting that it's still in memory only in one place for any reader who is curious